code/modules/projectiles/guns/syringe_gun.dm

```diff
@@ -68,6 +68,9 @@
 
 /obj/item/gun/syringe/attackby(obj/item/A, mob/user, params, show_msg = TRUE)
 	if(istype(A, /obj/item/reagent_containers/syringe))
+		if(istype(A, /obj/item/reagent_containers/syringe/lethal))
```

Comment:
Couldn't you just.. find a beaker and a regular syringe to sidestep this?
Maybe the lethal syringe could be made its own item instead?